### ISSUE TITLE
Resolutions lower than 1680px

### ISSUE THREAD

**Issue by nilsi** [sonyxperiadev/BacklogTool#42]
Lower the resolution under 1680px makes the interface look strange.
We should check how many of our users that is under this resolution and decide if we shall do a fix.

We should also look into what we can do when making the window smaller. (Also looks strange when making it small)

**Comment by GLundh** [comment]
Yeah. I think this needs to be prioritized. After upgrading the UI does look a bit strange. Especially when using really low resulutions (like when using a projector). It should certainly look good at 1024x768.
